src/js/components/Tabs.js

```diff
@@ -14,16 +14,34 @@ export default class Tabs extends Component {
     this._activateTab = this._activateTab.bind(this);
 
     this.state = {
-      activeIndex: props.initialIndex
+      activeIndex: props.initialIndex,
+      justify: props.justify
     };
   }
 
+  _addPropertyClass (classes, prefix, property, classProperty) {
```

Comment:
Why do we need this function, it seems a bit overkill for just adding `${CLASS_ROOT}--justify-${this.props.justify}`.